packages/graphql/src/schema/subscriptions/generate-subscription-types.ts

```diff
@@ -31,47 +34,77 @@ export function generateSubscriptionTypes({
 }) {
     const subscriptionComposer = schemaComposer.Subscription;
 
-    nodes.forEach((node) => {
-        const composeNode = schemaComposer.getOTC(node.name);
+    const eventTypeEnum = schemaComposer.createEnumTC(EventType);
 
+    nodes.forEach((node) => {
+        const eventPayload = generateEventPayloadType(node, schemaComposer);
+        const where = generateSubscriptionWhereType(node, schemaComposer);
         const subscribeOperation = node.rootTypeFieldNames.subscribe;
 
         const nodeCreatedEvent = schemaComposer.createObjectTC({
             name: `${node.name}CreatedEvent`,
             fields: {
-                [subscribeOperation.created]: composeNode,
+                event: {
+                    type: eventTypeEnum.NonNull,
+                    resolve: () => EventType.getValue("CREATE"),
+                },
+                [`created${node.name}`]: {
```

Comment:
:thinking: if we use node.name here we will have problems with underscores right?

Maybe we should expose `node.singular` or something like that to provide a standard name here